### PR TITLE
Upgrade to XWiki 9.8

### PR DESCRIPTION
**Pull request by vmassol**

### CODE REVIEW

File: library/xwiki [hunk 1]
```diff
@@ -6,13 +6,13 @@ GitCommit: fbc0fe3cbc07119c45ae8ea97c7e021d41e89b0a
 Directory: 8/mysql-tomcat
 
 Tags: 9, 9.7, 9-mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable
```
Comment (yosifkit) [inline library/xwiki]: Shouldn't the tags here and below be bumped to `9.8` as well? :wink:

Comment (vmassol) [reply library/xwiki]: grrr.... you're right of course... fixing

**Comment by vmassol** [comment]
Should be ok now. Thanks for checking!

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index 75bc247..b6fa78c 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -8,8 +8,8 @@ xwiki:8.4.5-postgres-tomcat
 xwiki:9
 xwiki:9-mysql-tomcat
 xwiki:9-postgres-tomcat
-xwiki:9.7
-xwiki:9.7-postgres-tomcat
+xwiki:9.8
+xwiki:9.8-postgres-tomcat
 xwiki:latest
 xwiki:lts
 xwiki:lts-mysql
diff --git a/xwiki_stable-postgres/Dockerfile b/xwiki_stable-postgres/Dockerfile
index c13a3f5..5dc282c 100644
--- a/xwiki_stable-postgres/Dockerfile
+++ b/xwiki_stable-postgres/Dockerfile
@@ -44,9 +44,9 @@ RUN apt-get update && \
 # Install XWiki as the ROOT webapp context in Tomcat
 # Create the Tomcat temporary directory
 # Configure the XWiki permanent directory
-ENV XWIKI_VERSION=9.7
+ENV XWIKI_VERSION=9.8
 ENV XWIKI_URL_PREFIX "http://maven.xwiki.org/releases/org/xwiki/platform/xwiki-platform-distribution-war/${XWIKI_VERSION}"
-ENV XWIKI_DOWNLOAD_SHA256 511edf81e9776b627d1a8aa200f6e377ab75c1821d7360d77a27092ad1ebf418
+ENV XWIKI_DOWNLOAD_SHA256 f416d4fccf6f766c902483b2cbc56236924c62ba2f5513128026c65f8d163ae2
 RUN rm -rf /usr/local/tomcat/webapps/* && \
   mkdir -p /usr/local/tomcat/temp && \
   mkdir -p /usr/local/xwiki/data && \
diff --git a/xwiki_stable/Dockerfile b/xwiki_stable/Dockerfile
index 32ef046..608b358 100644
--- a/xwiki_stable/Dockerfile
+++ b/xwiki_stable/Dockerfile
@@ -44,9 +44,9 @@ RUN apt-get update && \
 # Install XWiki as the ROOT webapp context in Tomcat
 # Create the Tomcat temporary directory
 # Configure the XWiki permanent directory
-ENV XWIKI_VERSION=9.7
+ENV XWIKI_VERSION=9.8
 ENV XWIKI_URL_PREFIX "http://maven.xwiki.org/releases/org/xwiki/platform/xwiki-platform-distribution-war/${XWIKI_VERSION}"
-ENV XWIKI_DOWNLOAD_SHA256 511edf81e9776b627d1a8aa200f6e377ab75c1821d7360d77a27092ad1ebf418
+ENV XWIKI_DOWNLOAD_SHA256 f416d4fccf6f766c902483b2cbc56236924c62ba2f5513128026c65f8d163ae2
 RUN rm -rf /usr/local/tomcat/webapps/* && \
   mkdir -p /usr/local/tomcat/temp && \
   mkdir -p /usr/local/xwiki/data && \
```

</details>

**Comment by yosifkit** [comment]
Build test of #3509; da19eb63 (`xwiki`):

```console

$ bashbrew build xwiki:8
Using bashbrew/cache:afe8e476b02156ec8055d778fafcc93c8f97982cb7aca060b56db30796cdf226 (xwiki:8)
Tagging xwiki:8
Tagging xwiki:8.4
Tagging xwiki:8.4.5
Tagging xwiki:8-mysql-tomcat
Tagging xwiki:mysql-tomcat
Tagging xwiki:lts-mysql-tomcat
Tagging xwiki:lts-mysql
Tagging xwiki:lts
Tagging xwiki:latest

$ test/run.sh xwiki:8
testing xwiki:8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build xwiki:9
Using bashbrew/cache:92be96d34b9be984e5b4d8a043673025c3f66f780a516df0769dd4fe7cb1fa00 (xwiki:9)
Tagging xwiki:9
Tagging xwiki:9.8
Tagging xwiki:9-mysql-tomcat
Tagging xwiki:stable-mysql-tomcat
Tagging xwiki:stable-mysql
Tagging xwiki:stable

$ test/run.sh xwiki:9
testing xwiki:9
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build xwiki:8-postgres-tomcat
Using bashbrew/cache:b4ab70ccfacbcdae748fb6367a85bf7c6d0eceddddbe662ce5ce9be7812b5240 (xwiki:8-postgres-tomcat)
Tagging xwiki:8-postgres-tomcat
Tagging xwiki:8.4-postgres-tomcat
Tagging xwiki:8.4.5-postgres-tomcat
Tagging xwiki:postgres-tomcat
Tagging xwiki:lts-postgres-tomcat
Tagging xwiki:lts-postgres

$ test/run.sh xwiki:8-postgres-tomcat
testing xwiki:8-postgres-tomcat
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build xwiki:9-postgres-tomcat
Using bashbrew/cache:126e10faf85f56a98b17bd12765056926cf36ad3de47fd6d391eada50cabac0f (xwiki:9-postgres-tomcat)
Tagging xwiki:9-postgres-tomcat
Tagging xwiki:9.8-postgres-tomcat
Tagging xwiki:stable-postgres-tomcat
Tagging xwiki:stable-postgres

$ test/run.sh xwiki:9-postgres-tomcat
testing xwiki:9-postgres-tomcat
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**